packages/ui-kit/src/components/Counter/Counter.types.ts

```diff
@@ -29,6 +29,8 @@ export interface CounterProps extends React.ComponentProps<'span'> {
     refetchInterval?: number
     /** Whether to retry on errors. */
     retry?: boolean
+    /** The Propel's API URL, defaults to the production API */
```

Comment:
Can you change this everywhere to this?
```suggestion
    /** This prop allows you to override the URL for Propel's GraphQL API. You shouldn't need to set this unless you are testing. */
```

Reply:
I just updated this for all the components